pages/logout.tsx

```diff
@@ -2,13 +2,18 @@ import { useEffect } from 'react'
 import { magic } from 'utils/magic'
 import Router from 'next/router'
 import Loader from 'components/Loader'
+import { encode as btoa } from 'base-64'
 
 export default function Logout() {
   useEffect(() => {
     const sayGoodbye = async () => {
       if (!magic) return false
       await magic.user.logout()
-      Router.push((await magic.user.isLoggedIn()) ? '/leaderboard' : '/login')
+      Router.push(
+        (await magic.user.isLoggedIn())
+          ? `/leaderboard`
+          : `/login?toast=${btoa('You have been logged out.')}`
```

Comment:
> Similar to logging in (which we have a ticket for), the menu pill doesn't update to remove your username when you log out. Since there's also not a "You have been logged out" toast, it looks like you get redirected to the login screen but are still logged in. I don't think both are needed, but without either there's potential for confusion.

@dguenther How does this work for you?

Reply:
That works 👍